src/BridgeModule.php

```diff
@@ -160,6 +160,8 @@ class BridgeModule extends Module implements BootstrapInterface
 
     public $settingsCacheKey = 'bridge_settings';
 
+    public $layout = '@bridge/views/layouts/main';
```

Comment:
It would be cool to have some comment to this field that would describe the use-case, @altynbek07 do you have time for that?

Reply:
@naffiq done